lib/deposit.dart

```diff
@@ -199,6 +199,7 @@ class _CryptoDepositsScreenState extends State<CryptoDepositsScreen> {
     var res = await beCryptoDepositRecipient(
         widget.asset.symbol, widget.l2Network?.symbol, amount);
     Navigator.pop(context);
+    Navigator.pop(context);
```

Comment:
We dont want this here. we want it after `MaterialPageRoute(
                builder: (context) => CryptoDepositNewScreen(widget.asset,
                    widget.l2Network, recipient, widget.websocket)))` (line 205/6) *IF* it was successful in creating a new deposit